EarlyBinding/src/EXPRESS/EXPRESSEntity.h

```diff
@@ -31,25 +29,26 @@
 
 OIP_NAMESPACE_OPENINFRAPLATFORM_EARLYBINDING_BEGIN
 
+class EXPRESSModel;
 
-class EXPRESSEntity : public EXPRESSObject {
+OIP_EARLYBINDING_EXTERN
+class
+OIP_EARLYBINDING_API_EXPRESS

```

Comment:
Why both OIP_EARLYBINDING_EXTERN and OIP_EARLYBINDING_API_EXPRESS?

Reply:
_EXTERN stands for `extern "C"`, _API for `dllexport/import`